switchapi/es2k/lnw_v3/switch_pd_p4_name_routing.h

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright (c) 2013-2021 Barefoot Networks, Inc.
+ * Copyright 2022-2023 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __SWITCH_PD_P4_NAME_ROUTING_MAPPING__
+#define __SWITCH_PD_P4_NAME_ROUTING_MAPPING__
+
+#ifdef __cplusplus
+extern "C" {
+#endif /* __cplusplus */
+
+#define MAC_BASE 0
```

Comment:
These definitions don't belong in `switch_pd_p4_name_routing.h`. They have nothing to do with table names.

Use a different header file if they're shared by two or more files; make them local to the file that uses them if it's the only one.